server/routes.js

```diff
@@ -1,18 +1,24 @@
-var bodyParser = require('body-parser');
+var bodyParser = require("body-parser");
+import webhooks from "/server/slack/webhooks.js";
 
 Picker.middleware(bodyParser.json());
 
-Picker.route('/callback/hangout/:_id', function(params, req, res) {
-  res.setHeader('Access-Control-Allow-Origin', '*');
-  res.setHeader('Access-Control-Allow-Methods', 'PUT');
-  res.setHeader('Access-Control-Allow-Headers', 'Content-Type');
-	//add in some checking ot make sure re.body.hangoutUrl is a hangout URL
-  Hangouts.update(params._id, { $set: { url: req.body.hangoutUrl }}, function(err, result){
-  	if (err) {
-  		res.statusCode = 404;
-  	} else {
-  		res.statusCode = 200;
-  	}
-  	res.end();
+Picker.route("/callback/hangout/:_id", function(params, req, res) {
+  res.setHeader("Access-Control-Allow-Origin", "*");
+  res.setHeader("Access-Control-Allow-Methods", "PUT");
+  res.setHeader("Access-Control-Allow-Headers", "Content-Type");
+  //add in some checking ot make sure re.body.hangoutUrl is a hangout URL
+  Hangouts.update(params._id, { $set: { url: req.body.hangoutUrl } }, function(
+    err,
+    result
+  ) {
+    if (err) {
+      res.statusCode = 404;
```

Comment:
404 is reserved for page not found. not sure, but I think for a server side error, we should use 500

Reply:
@sebbel +1